tester/reporter_test.go

```diff
@@ -152,6 +168,12 @@ func TestJSONReporter(t *testing.T) {
 			Fail:    true,
 			Trace:   getFakeTraceEvents(),
 		},
+		{
+			Package: "data.foo.bar",
+			Name:    "todo_test_qux",
+			Skip:    true,
+			Trace:   nil,
```

Comment:
[nit] This is probably the zero value if you'd leave off the `Trace` field -- but it doesn't hurt to be explicit 😃

Reply:
Thanks for the reminder! In this case I agree that it doesn't hurt to be specific and show that it's not just an omission that the trace is missing.